CHANGELOG.md

```diff
@@ -2,7 +2,8 @@ CHANGELOG
 =========
 
 ## HEAD (Unreleased)
-_(none)_
+* Upgrade to v6.1.4 of the Kong Terraform Provider
+  **Breaking Change:** `kong.JwtToken` has been renamed to `kong.ConsumerJwtToken` due to a change in the upstream provider
```

Comment:
I don't see this change in the schema (and schema-tools)... I only see a net-new resource called "ConsumerJwtAuth" (not "ConsumerJwtToken")

Reply:
You are correct - this means that it wasn't mapped correctly before :) So this is just a simple upgrade - I removed the breaking change line